apps/portals/ampals/src/pages/HomePage.tsx

```diff
@@ -84,6 +86,71 @@ export default function HomePage() {
         />
       </SectionLayout>
       <AMPALSPublishingRequirements />
+      <SectionLayout
+        title={'Featured Datasets'}
+        centerTitle
+        ContainerProps={{
+          sx: {
+            maxWidth: '100% !important',
+            padding: { xs: '40px', lg: '60px 80px 80px 80px' },
+            h2: {
+              lineHeight: 'normal',
+              margin: '0',
+            },
+          },
+        }}
+      >
+        <FeaturedDataTabs
+          sql={datasetsSql}
+          rgbIndex={3}
+          configs={[
+            {
+              title: 'Datasets',
+              icon: 'DATASET',
+              explorePagePath: '/Explore/Datasets',
+              exploreObjectType: 'Datasets',
+              plotsConfig: {
+                configs: [
+                  {
+                    title: 'Gene Expression Omnibus',
+                    description: '',
+                    facetsToPlot: ['dataType'],
+                    selectFacetColumnName: 'source',
+                    selectFacetColumnValue: 'Gene Expression Omnibus',
+                    detailsPagePath:
+                      '/Explore/Datasets?QueryWrapper0=%7B"sql"%3A"SELECT+*+FROM+syn66496326"%2C"limit"%3A25%2C"selectedFacets"%3A%5B%7B"concreteType"%3A"org.sagebionetworks.repo.model.table.FacetColumnValuesRequest"%2C"columnName"%3A"source"%2C"facetValues"%3A%5B"Gene+Expression+Omnibus"%5D%7D%5D%7D',
```

Comment:
Do you think it would be a good idea to build this as a Query JSON object before we serialize it?  May reduce the likelihood these links break without realizing if the base dataset SQL changes.